agent/tests/agent-wrapper.php

```diff
@@ -95,11 +98,15 @@
     });
 } else {
     call_user_func(static function () use ($listenOn, $browserFactory, $serverResolver, $loop) {  // @phpstan-ignore closure.unusedUse, closure.unusedUse, closure.unusedUse, closure.unusedUse
+        $basePath = __DIR__.'/../build';
```

Comment:
This is needed to have a correct base path when we are not running the phar. The phar uses special logic to determine the base path. This just lets us override it when calling the agent directly not via the phar.